### PR TITLE
Move Csv-AppController as BaseModuleController (task #7507)

### PR DESCRIPTION
**Pull request by eliamariutti**
Moving the controller and related action views in the app domain to start making the project-template less depend on the CsvMigration plugin.

### CODE REVIEW

File: src/Template/Element/View/associated.ctp [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        'var url = document.location.toString();
+            if (matches = url.match(/(.*)(#.*)/)) {
+                $(".nav-tabs a[href=\'" + matches["2"] + "\']").tab("show");
+                history.pushState("", document.title, window.location.pathname + window.location.search);
+            }
+        ',
+        ['block' => 'scriptBottom']
+    ); ?>
+    <div class="nav-tabs-custom">
+        <?= $this->element('CsvMigrations.Associated/tabs-list', [
```
Comment (softius) [inline src/Template/Element/View/associated.ctp]: @eliamariutti i believe this must load from app level.

File: src/Template/Element/View/associated.ctp [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            'table' => $table, 'associations' => $associations
+        ]); ?>
+        <?= $this->element('CsvMigrations.Associated/tabs-content', [
```
Comment (softius) [inline src/Template/Element/View/associated.ctp]: @eliamariutti i believe this must load from app level.

File: src/Template/Element/View/view.ctp [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+if (!$this->request->getQuery('embedded')) : ?>
+<section class="content-header">
+    <div class="row">
+        <div class="col-xs-12 col-md-6">
+            <h4><?= $options['title'] ?></h4>
+        </div>
+        <div class="col-xs-12 col-md-6">
+            <div class="pull-right">
+            <div class="btn-group btn-group-sm" role="group">
+                <?= $this->element('CsvMigrations.Menu/view_top', [
```
Comment (softius) [inline src/Template/Element/View/view.ctp]: @eliamariutti i believe this must load from app level.

File: src/Template/Element/View/view.ctp [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+            </div>
+        <?php endforeach; ?>
+        </div>
+    </div>
+    <?php
+    if (empty($embeddedFields)) {
+        continue;
+    }
+
+    echo $this->element('CsvMigrations.Embedded/fields', [
```
Comment (softius) [inline src/Template/Element/View/view.ctp]: @eliamariutti i believe this must load from app level.

**Comment by anvyst** [comment]
Great job, @eliamariutti !

**Pull request merged**